File: Easy/goat-latin.py
class Solution:     # 45ms
    def toGoatLatin(self, sentence: str) -> str:
        sentence = sentence.split()
        ret = ""
        for i in range(len(sentence)):
            first = sentence[i][0]
            if first.lower() not in 'aeiou':
                sentence[i] = sentence[i][1:] + sentence[i][0]
            sentence[i] += 'maa'
            for _ in range(i):
                sentence[i] += 'a'
            ret += sentence[i] + ' '
                
        return ret[:-1]

File: Easy/goat-latin.java
class Solution {        // 8ms
    public String toGoatLatin(String sentence) {
        String ret = "";
        String[] arr = sentence.split(" ");
        
        for(int i = 0; i < arr.length; i++){
            if(!"aeiouAEIOU".contains("" + arr[i].charAt(0))){
                ret += arr[i].substring(1);
                ret += arr[i].charAt(0);
            } else {
                ret += arr[i];
            }
            ret += "maa";
            for(int j = 0; j < i; j++){
                ret += 'a';
            }
            ret += ' ';
        }
        return ret.substring(0, ret.length() - 1);
    }
}